tests/fixtures/kv12ntfs/output/lines.txt

```diff
@@ -1,4 +1,4 @@
 line_id,line_code,line_name,forward_line_name,backward_line_name,line_color,line_text_color,line_sort_order,network_id,commercial_mode_id,geometry_id,line_opening_time,line_closing_time
-prefix:2029,29,"Slagharen, Centrum - Slagharen, Attractiepark","Slagharen, Centrum - Slagharen, Attractiepark","Dedemsvaart, Busstation - Dedemsvaart, Adelaarweg",,,,prefix:SYNTUS,Bus,,,
+prefix:2029,29,"Hoogeveen, Willemskade - Hoogeveen, Willemskade","Hoogeveen, Willemskade - Hoogeveen, Willemskade","Dedemsvaart, Busstation - Dedemsvaart, Adelaarweg",,,,prefix:SYNTUS,Bus,,,
```

Comment:
We used the name of the stops previously, and now the name of the stopareas. 
Which gives this strange result, since the 2 stoppoints are in the same stoparea.
We will check on the production data set before, because it can be considered as a regression finally.

Reply:
I wouldn't worry to check on production because we know this PR is a regression already: we have changed the way we calculate the `route.name`. I just happened to catch a bug on the way. But saying that, I realize I should update the `Cargo.toml` to `0.13.0`.

Reply:
The api is not broken. To me it's a fix

Reply:
Just 2 cases in Production with the refactoring :
`Ede, Station Ede-Wageningen - Ede, Ziekenhuis Gelderse Vallei`
--> `Ede, Station Ede-Wageningen - Ede, Station Ede-Wageningen`
`Enschede, Centraal Station - Hengelo, Centraal Station`
--> `Hengelo, Centraal Station - Hengelo, Centraal Station`
Ok for me :+1: , we will see depending on possible returns.

Reply:
@datanel, we're completely changing the way this field is generated so the output format is necessarily different from version `0.12.0`. You would consider this a non-breaking change? Should I move back to `0.12.1`?

Reply:
We can consider that was a bug

Reply:
I just changed it back to `0.12.1`.